### ISSUE TITLE
[Req] Define a primary player

### ISSUE THREAD

**Issue by FlashHit** [EmulatorNexus/Fusion-UI#9]
It would be nice to be able to choose a primary player out of the 4 possible players you can have within 1 account.
Joining a game with vu://join/Server-guid would then choose the primary player automatically.

https://github.com/EmulatorNexus/VeniceUnleashed/issues/630

**Comment by kapraran** [comment]
I like this.

Is it fine to use the account storage for storing the primary player? Or it may need a more special place? @OrfeasZ

**Comment by OrfeasZ** [comment]
Account storage is fine for this. VU will hydrate the account storage around the same time (either directly before or directly after) as it populates the player list, so that should work pretty well.

**Comment by kapraran** [comment]
Good

But how it should work? Like if you have a primary player it should only bypass the screen if there's a pending connection?

Because if it will always bypass the soldier selection screen, then you wont be able to change a soldier again, since there isn't a "soldier logout" currently https://github.com/EmulatorNexus/Fusion-UI/issues/7

**Comment by OrfeasZ** [comment]
I think that makes sense for now. If / when switching soldiers is implemented we can look into adjusting it accordingly.

**Comment by kapraran** [comment]
Sure

**Comment by FlashHit** [comment]
> Joining a game with vu://join/Server-guid would then choose the primary player automatically.

that’s the main purpose for me

**Comment by kapraran** [comment]
How to tell if there's a pending connection going on when you are on Soldier selection screen?

Do you dispatch SET_PENDING_SERVER from VU code, after soldier login? Because I dont see any check happening in js, when main menu is mounted.

**Comment by OrfeasZ** [comment]
@kapraran yes, `SET_PENDING_SERVER` is sent at most a second after you've logged in with a soldier. I'll see if I can move it somewhere earlier.
